examples/gallery/maps/roads.py

```diff
@@ -0,0 +1,42 @@
+"""
+Roads
+-----
+The :meth:`pygmt.Figure.plot` method allows to plot geographical data which is
+stored in a :class:`geopandas.GeoDataFrame` object.
+"""
+
+import geopandas as gpd
+import pygmt
+
+# Read shapefile data using geopandas
+gdf = gpd.read_file(
+    "http://www2.census.gov/geo/tiger/TIGER2015/PRISECROADS/tl_2015_15_prisecroads.zip"
+)
+# The dataset contains different road types listed in the RTTYP column,
+# here we select the following ones to plot:
+roads_common = gdf[gdf.RTTYP == "M"]  # Common name roads
+roads_state = gdf[gdf.RTTYP == "S"]  # State recognized roads
+roads_interstate = gdf[gdf.RTTYP == "I"]  # Interstate roads
+
+fig = pygmt.Figure()
+
+# Define target region around O'ahu (Hawai'i)
+region = [-158.3, -157.6, 21.2, 21.75]  # minx, maxx, miny, maxy
+
+fig.basemap(
+    region=region,
+    projection="M12c",
+    frame=["af", 'WSne+t"Main roads of O`ahu (Hawai`i)"'],
```

Comment:
Not sure if using backticks (`) is ok here. It looks like this:

![image](https://user-images.githubusercontent.com/23487320/132124184-c7f68a84-34b0-4849-bc76-512f5e61b964.png)

I think it should be pointing in the other direction (towards the left) instead? Maybe we should just leave this out, or find a way to escape the (') somehow.

Reply:
Agree with you @weiji14. So far I found no solution to escape in the right way. Thus, I removed it.